pkg/configuration/constants.go

```diff
@@ -36,6 +36,9 @@ const (
 	RAW_CMD_ARGS                   string = "internal_raw_cmd_args"
 	WEB_APP_URL                    string = "internal_snyk_app"
 	MAX_THREADS                    string = "internal_max_thread_count"
+	CONFIG_CACHE_DURATION          string = "internal_config_cache_ttl"
+	CONFIG_CACHE_CLEANUP_INTERVAL  string = "internal_config_cache_cleanup_interval"
```

Comment:
question: I wonder why a cleanup interval needs to be specified? Can we not rely solely on the TTL? Is this an implementation detail of the caching package?

IMO it makes sense to define a cache size and evict items based on LRU for example.

Reply:
I was thinking that things like LRU and cleanup intervals are methods to manage the size of the cache, and so defining a limitation on the cache, like max memory size might make more sense.